src/components/schedule-page/validation-drawer/error-list-item.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Button } from "../../common-components";
+import { ScheduleErrorMessageModel } from "../../../common-models/schedule-error-message.model";
+
+interface Options {
+  error: ScheduleErrorMessageModel;
+}
+
+export default function ErrorListItem({ error }: Options): JSX.Element {
+  return (
+    <div className="error-list-item">
+      <div className="red-rectangle"></div>
+      <div className="error-title">
+        <p className="error-title-content">10 października{error.day}</p>
```

Comment:
@pectom , czy wprowadzamy nagłówki w tym `MR`?

Reply:
Nie trzeba, niech to wejdzie w kolejnym.